deployment/helm/resource-policy/topology_aware_values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for nri-plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+image:
+  name: ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware
+  # tag, if defined will use the given image tag, otherwise Chart.AppVersion will be used
+  tag: unstable
+  pullPolicy: Always
+container:
+  name: nri-resource-policy-topology-aware
+
+namespace: kube-system
+
+configMap:
+  data:
+    ReservedResources:
+      cpu: 750m
+
+spec:
+  containers:
+    ports:
+    - containerPort: 8891
+      protocol: TCP
+      name: metrics
```

Comment:
Now taking a second look at this, I'd simplify the values a lot and move all the spec.containers stuff just directly to the daemonset template. These nested lists and args etc are very awkward for the user to set from the command line, think like `helm install --set spec.containers.ports[0].containerPort=9876`.

I'd prefere a lot "flatter" structure, e.g. with top-level key like `metricsPort` or `metrics.port` directly.

Keep it simple and add stuff when required.

WDYT?

Reply:
good point. Fixed